google/pubsub/v1/pubsub.proto

```diff
@@ -417,7 +417,7 @@ message Topic {
   // must not start with `"goog"`.
   string name = 1 [
       (google.api.field_behavior) = REQUIRED,
-      (google.api.resource).path = "projects/{project_id}/topics/{topic_id}"];
+      (google.api.resource).path = "projects/{project}/topics/{topic}"];
```

Comment:
Generally these should be `project_id` and `topic_id` but I think they need to stay project and topic to keep compatibility with the current gapic config.

Reply:
As commented on in https://github.com/googleapis/api-common-protos/pull/24/files the C# generator currently auto-adds the `id` suffix if it's missing, so C# won't be broken by using `project_id` and `topic_id` here.
@andreamlin How does Java handle this?